.gitignore

```diff
@@ -127,3 +127,9 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# DS store
+.DS_Store
+
+# skip extension private key
+hw1/chris_extension.pem
```

Comment:
can change to `*.pem` to be generic so future private keys aren't staged without needing to update the gitignore for all